src/wilson/dom.cljs

```diff
@@ -78,6 +79,38 @@
   ([ks rows]
    (table ks rows {})))
 
+
+(defn sorted-table
+  ([ks rows opts]
+   (let [state (r/atom {:sort-key (first ks)
+                        :sort-order :asc})
+         state-deref @state
+         sort-fns (or (:sort-fns opts)
```

Comment:
Yep. Clicking on each `th` in the table will try to trigger a function under key that column displays data for in `sort-fns` map. If that key is not in map, a function under `:default` will be used.
So in `sorted-table` I just set a simple `:default` sorting fn (or let the user pass their own)